src/components/OperationsBuilder.js

```diff
@@ -75,7 +82,10 @@ let operation = (ops, index, dispatch) => {
           maxLength="2"
           className="TransactionOpMeta__order__input" />
       </div>
+      <div>
       {removeLink}
+      {duplicateLink}
```

Comment:
What if we swapped these two buttons (duplicate would always be on top)? Would it be better for the user if the button order didn't change? 🤔

Reply:
Thanks. Makes a lot of sense